### DISCUSSION TITLE
Tag value size is sometimes a problem

### DISCUSSION THREAD

**Discussion by matkoniecz** [osmlab/osm-data-model#13] [Study/Solutions]
> On the other hand, for tag values they might be too short for some of things we are doing with them.39 But we have been living with them for a long time and haven’t seen any problems

Complex `opening_hours` can run into such limits, and there are repeated questions from mappers how to handle https://wiki.openstreetmap.org/wiki/Key:inscription not fitting entire inscription.

See https://github.com/openstreetmap/openstreetmap-website/issues/2025 for example.

Likely not worth touching in isolation, but in case of backward incompatible changes it should be at least considered.

Or at least not dismissed with claim that this problem is not existing at all.

EDIT: for example see https://www.openstreetmap.org/note/3331071 (I was not trying to find it! I was just going through https://resultmaps.neis-one.org/osm-notes-country-custom?c=Poland&query=open note list )

**Comment by joto** [comment]
Well, I wouldn't consider those cases "problems". Maybe I should have written "larger problems" or so. Anything we do will always have somebody running against some limits, so I'd consider a few (and contested) cases in 18 years "not a problem".

**Comment by CorruptComputer** [comment]
I have had my changeset comments cut off quite often, I think raising this limit would be a good change.

**Reply by matkoniecz** [reply]
I wanted to comment that it is out of scope, but actually it is covered!

> This is about the OSM data model, the nodes, ways, relations, tags, changesets.
